FileStorage.Client/Program.cs

```diff
@@ -38,9 +38,40 @@ static void Main(string[] args)
             sw.Stop();
 
             Console.WriteLine(Environment.NewLine + "Transferred " + fi.Length + " bytes in " + sw.ElapsedMilliseconds);
+
+
+            string filenameRead = @"testPayloadRead.txt";
+            GetFile(filename, filenameRead);
+
             Console.ReadLine();
         }
 
+
+        private static void GetFile(string getFile, string saveFile)
+        {
+            HttpWebRequest webRequest = (HttpWebRequest) HttpWebRequest.Create("http://localhost:8081/Api/File/" + getFile);
+            webRequest.Method = "GET";
+
+            (webRequest as WebRequest).Headers.Add(HttpRequestHeader.Authorization,
+                "Bearer 5c5d3b905c00fdc9817809e324fbe4ab");
+            (webRequest as WebRequest).ContentLength = 0;
```

Comment:
But what we do need is to add/support the headers from the spec: https://msdn.microsoft.com/en-us/library/azure/dn194274.aspx (see the RequestHeaders section)